docs/languages/en/migration/namespacing-old-classes.rst

```diff
@@ -3,7 +3,7 @@
 Namespacing Old Classes
 =======================
 
-ZF2's minimal version is PHP 5.3.  The most notable feature of PHP 5.3 is
+ZF2's minimal version is PHP 5.4.  The most notable feature of PHP 5.3 is
```

Comment:
This is odd, we are telling them the minimal version is 5.4 but then talking about 5.3 features.